app/spec/lib/bk/compat/circleci_spec.rb

```diff
@@ -21,3 +21,143 @@
     end
   end
 end
+
+RSpec.describe BK::Compat::CircleCISteps::DockerInstaller do
```

Comment:
if you remove the class, this should be removed as well. If not, I'd suggest moving it to its own spec file